pkg/rancher-desktop/backend/steve.ts

```diff
@@ -46,7 +47,13 @@ export class Steve {
 
     const osSpecificName = /^win/i.test(os.platform()) ? 'steve.exe' : 'steve';
     const stevePath = path.join(paths.resources, os.platform(), 'internal', osSpecificName);
+    const env = process.env;
```

Comment:
Do you want to duplicate `process.env`? Currently it modifies the actual env in the main process, so we'll accidentally have `KUBECONFIG` set (e.g. when we try to update it next time).